utils.sh

```diff
@@ -685,4 +685,15 @@ use_podman_registry() {
   return 1
 }
 
+use_quay_registry() {
+  # return true if using the local registry and the backend is podman
```

Comment:
The comment seems to be wrong.
Minor: it'd be nice to remove the duplication and have just a single function, ie `use_registry "podman"` or `use_registry "quay"`. The argument could be optionally checked if not empty, so that the same function could be reused in `02_configure_host.sh` line 72, to remove also that duplication

Reply:
Good idea. Done.